server/protocols/webadmin/webadmin-mailrepository/src/main/java/org/apache/james/webadmin/service/MailRepositoryStoreService.java

```diff
@@ -56,6 +56,9 @@ public MailRepositoryStoreService(MailRepositoryStore mailRepositoryStore) {
             .collect(Guavate.toImmutableList());
     }
 
+    public MailRepository createMailRepository(String repositoryUrl) throws MailRepositoryStore.MailRepositoryStoreException {
+        return mailRepositoryStore.select(repositoryUrl);
```

Comment:
really ? don't we have a proper `create` method ?

Reply:
Nope, we should add it.